src/pages/partners/affiliate-ib/_faq-data.tsx

```diff
@@ -462,10 +462,8 @@ const AffiliateReferralTools = () => (
                 type: 'paragraph-2',
             }}
         >
-            <Localize
-                translate_text="We have a tried-and-tested selection of referral tools, including banners, videos, reviews, and text ads. If you’d like certain tools to be customised to your site requirements, please contact your account manager at <0>affiliates@deriv.com</0>."
-                components={[<StyledLink href="mailto:affiliates@deriv.com" key={0} />]}
-            />
+            <Localize translate_text="We have a tried-and-tested selection of referral tools, including banners, videos, reviews, and text ads. If you’d like certain tools to be customised to your site requirements, " />
+            <LiveChatLinkText text="contact us via live chat." />
```

Comment:
```suggestion
            <LiveChatLinkText text="_t_contact us via live chat._t_" />
```
since you used `text` directly in `LiveChatLinkText`, you have to wrap the text with `_t_`.